app/src/main/java/com/example/fbufinalapp/EditDestinationActivity.java

```diff
@@ -93,6 +93,15 @@ protected void onCreate(Bundle savedInstanceState) {
                 query.getInBackground(getIntent().getStringExtra("destinationId"), (object, e) -> {
                     if (e == null) {
                         editingDestination = object;
+                        Date date = editingDestination.getDate();
+
+                        SimpleDateFormat dateFormatter = new SimpleDateFormat("MMM dd, yyyy");
```

Comment:
is `MMM` a typo?

Reply:
Yes, thank you for catching that! I've fixed it now :)